### PR TITLE
Write Entry API guide about price info

### PR DESCRIPTION
**Pull request by grubolsch**
### Added

- Added examples on PUT /events/:eventId/price-info
- Added examples on PUT /place/:placeId/price-info
- Added extra page on price info structure

Screenshot 2023-10-12 at 13 22 46
Screenshot 2023-10-12 at 13 23 12
Screenshot 2023-10-12 at 13 23 05

---

Ticket: https://jira.uitdatabank.be/browse/III-5455

### CODE REVIEW

File: projects/uitdatabank/docs/entry-api/shared/price-info.md [hunk 1]
```diff
@@ -1,8 +1,51 @@
...
+  "properties": {
+    "priceInfo": {
+       "$ref": "./../../../models/common-priceInfo.json"
+     }
+  }
+}
+```
+
+### What is the category base?
+The 'base' category the primary pricing baseline for events or places. This price will later be used for calculating discounts (eg. Uitpas tariff).
```
Comment (JonasVHG) [inline projects/uitdatabank/docs/entry-api/shared/price-info.md:27]: 2 remartks here:
* Always keep the branding in mind, it's UiTPAS 😉
* Since not all events have an UiTPAS-discount I would rephrase this.
```suggestion
The 'base' category the primary pricing baseline for events or places. If applicable, this price is also used to calculate the UiTPAS discount.```

**Review by JonasVHG — CHANGES_REQUESTED** [review]
Looks good, 
just one sentence I would rephrase and the incorrect branded casing of UiTPAS should be fixed.

**Comment by grubolsch** [comment]
All feedback is processed.

**Review by corneelwille — APPROVED** [review]

File: projects/uitdatabank/docs/entry-api/shared/price-info.md (hunk 1 above)
Comment (grubolsch) [reply projects/uitdatabank/docs/entry-api/shared/price-info.md]: Should be fine now, but I kept the casing for this part, because the casing in the code is also just "Uitpas" and not "UiTPAS". It would be confusing to use the wrong casing here.

**Review by JonasVHG — APPROVED** [review]
